src/pages/Home/Home.jsx

```diff
@@ -29,6 +30,7 @@ const Home = ({openLoginModal, windowWidth}) => {
             className="login-button"
             handleOnClick={openLoginModal}
           />
+          <h4 className="error">{error ? error : ''}</h4>
```

Comment:
Let's also conditionally render the entire `h4` tag so that we don't have an empty `h4` when there is no `error` message here too.